<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QDialog" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>623</width>
    <height>623</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>623</width>
    <height>623</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>623</width>
    <height>623</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>로그인</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::RightToLeft</enum>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="login_btn">
   <property name="geometry">
    <rect>
     <x>92</x>
     <y>490</y>
     <width>443</width>
     <height>71</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>443</width>
     <height>71</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>443</width>
     <height>71</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 0);
background-color: rgb(66, 195, 255);</string>
   </property>
   <property name="text">
    <string>LOGIN</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="userid">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>285</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>181</width>
     <height>31</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>181</width>
     <height>31</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="placeholderText">
    <string>USER ID</string>
   </property>
  </widget>
  <widget class="QComboBox" name="combo_02">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>427</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>181</width>
     <height>31</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>181</width>
     <height>31</height>
    </size>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <item>
    <property name="text">
     <string>LANGUAGE</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>한국어</string>
    </property>
   </item>
  </widget>
  <widget class="QTextEdit" name="psswd">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>357</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>181</width>
     <height>31</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>181</width>
     <height>31</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="placeholderText">
    <string>PASSWORD</string>
   </property>
  </widget>
  <widget class="QComboBox" name="combo_01">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>215</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>181</width>
     <height>31</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>181</width>
     <height>31</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <item>
    <property name="text">
     <string>COMPANY</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>본사</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
